theme/src/components/head.js

```diff
@@ -15,6 +15,12 @@ function Head(props) {
       <meta property="og:description" content={description} />
       <meta property="og:image" content={siteMetadata.imageUrl} />
       <meta property="twitter:card" content="summary_large_image" />
+      {siteMetadata.primerTracking ? (
```

Comment:
Does this use cookies at all? If so do we need to add a cookie notification when siteMetadata.primerTracking is true?